localgov_alert_banner.install

```diff
@@ -12,14 +12,21 @@ function localgov_alert_banner_uninstall() {
 
   // Remove configuration.
   $configs = [
-    "block.block.views_block__alert_banner_alert_banner_block",
-    "core.entity_form_display.alert_banner.alert_banner.default",
-    "core.entity_view_display.alert_banner.alert_banner.default",
-    "flag.flag.localgov_put_live",
-    "system.action.flag_action.localgov_put_live_flag",
-    "system.action.flag_action.localgov_put_live_unflag",
-    "user.role.emergency_publisher",
-    "views.view.alert_banner",
+    "block.block.views_block__alert_banner_alert_banner_block.yml",
+    "core.entity_form_display.localgov_alert_banner.localgov_alert_banner.default.yml",
+    "core.entity_view_display.localgov_alert_banner.localgov_alert_banner.default.yml",
+    "field.field.localgov_alert_banner.localgov_alert_banner.link.yml",
+    "field.field.localgov_alert_banner.localgov_alert_banner.short_description.yml",
+    "field.field.localgov_alert_banner.localgov_alert_banner.type_of_alert.yml",
+    "field.storage.localgov_alert_banner.link.yml",
+    "field.storage.localgov_alert_banner.short_description.yml",
+    "field.storage.localgov_alert_banner.type_of_alert.yml",
+    "flag.flag.localgov_put_live.yml",
+    "localgov_alert_banner.localgov_alert_banner_type.localgov_alert_banner.yml",
+    "system.action.flag_action.localgov_put_live_flag.yml",
+    "system.action.flag_action.localgov_put_live_unflag.yml",
+    "user.role.emergency_publisher.yml",
+    "views.view.alert_banner.yml",
```

Comment:
These should not include the `.yml` at the end.

Reply:
Doh. Updated.